std/net/curl.d

```diff
@@ -313,7 +313,11 @@ version(unittest)
 version(StdDdoc) import std.stdio;
 
 // Default data timeout for Protocols
-private enum _defaultDataTimeout = dur!"minutes"(2);
+private auto _defaultDataTimeout() {
+    import core.time : dur;
+    static res = dur!"minutes"(2);
```

Comment:
`static` is now going to make it take up space in TLS (and be initialized for each new thread), for something that should be a manifest constant...